pkg/rules/rulespb/rpc.proto

```diff
@@ -26,7 +27,30 @@ option (gogoproto.goproto_sizecache_all) = false;
 service Rules {
     /// Rules has info for all rules.
     /// Returned rules are expected to include external labels.
-    rpc Rules(RulesRequest) returns (stream RulesResponse);
+    rpc Rules(RulesRequest) returns (stream RulesResponse){};
+
+    /// Gets a rule recording.
+    rpc RulesRecord(RuleRecordRequest) returns (RecordingRule){
```

Comment:
Do we need this? 🤔

Reply:
Do you mean  `rpc Rules(RulesRequest)` ? This protobuf definition is originally for gRPC, and this rpc is for gRPC. I will define REST API in another new file instead of modifying this.